backend/pom.xml

```diff
@@ -16,6 +16,26 @@
     <properties>
         <java.version>11</java.version>
     </properties>
+
+    <profiles>
+        <profile>
+            <id>local</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <spring.profiles.active>local</spring.profiles.active>
+            </properties>
+        </profile>
+        <profile>
+            <id>docker</id>
```

Comment:
Since this can be reused and it is not tightly-coupled with docker you can rename it to lets say to no-test or something like that

Reply:
I thought we should have a dedicated profile for Docker, though right now it just has no-tests. I'll rename it.